src/nomad_simulations/properties/energies.py

```diff
@@ -77,3 +78,502 @@ def __init__(
+        super().normalize(archive, logger)
+
+####################################################
+# List of classical energy contribuions
+####################################################
+
+class PotentialEnergy(ClassicalEnergy):
+    """
+    Physical property section describing the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+class IntermolecularEnergy(ClassicalEnergy):
+    """
+    Physical property section describing all intramolecular contributions to the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+class VDWEnergy(ClassicalEnergy):
+    """
+    Physical property section describing the van der Waals contributions to the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+class ElectrostaticEnergy(ClassicalEnergy):
+    """
+    Physical property section describing all electrostatic contributions to the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+class ElectrostaticShortRangeEnergy(ClassicalEnergy):
+    """
+    Physical property section describing short-range electrostatic contributions to the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+class ElectrostaticLongRangeEnergy(ClassicalEnergy):
+    """
+    Physical property section describing long-range electrostatic contributions to the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class IntramolecularEnergy(ClassicalEnergy):
+    """
+    Physical property section describing all intramolecular contributions to the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class BondEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from bond interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class AngleEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from angle interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class DihedralEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from dihedral interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
```

Comment:
Just to be clear I paste my explanation from above:

In classical simulations it is common to store the contribution to the total energy (usually per particle) from a particular term of the force field. E.g., you may want to know the total energy on a particle in a configuration coming from some dihedral potential only. I think this is why you thought there is a confusion between energy and potential. The potential is indeed stored within the FF section of method. But the corresponding energy is the evaluation of that potential or set of potentials for a particular configuration.

Reply:
> the contribution to the total energy (usually per particle) from a particular term of the force field

Yes, in QM we do the same with Hamiltonian (which may encode more than just the potential). Up to a certain extent, QM codes also report the energy decomposition as such, though not always. The crux is that this is model-specific, by definition. I.e. you're generating a semantic language to describe all physical models. This may be fine, but should then tie back to the `method` section.

What I was suggesting is to focus instead on _physical processes_. I think those are more interoperable and of general interest i. E.g. _kinetic energy_ which breaks apart into _translational_, _rotational_, and _vibrational_ is relevant for the virial theorem, entropy, etc. I would therefore make these into subsections.

> total energy on a particle in a configuration

Well, this is firstly partitioned by the structure, so you should decide on a mapping there. The rest is method specific data, so I would see to tie to the method definition. Still, one can also analyze (but maybe not easily extract) the dihedral contribution and others in a rotational and vibrational mode.